### PR TITLE
42 mobile header menu

### PR DESCRIPTION
**Pull request by perezrei**
This is the burger menu for the mobile version.
I pushed resized images in homepage and the logo in the header

### CODE REVIEW

File: client/components/header-component.js [hunk 1]
```diff
@@ -2,22 +2,44 @@ import AvatarDropdown from "./avatar-dropdown-component.js";
...
+              data-bs-target="#header-menu-burger">
+            <span class="header-menu-close-line"></span>
+            <span class="header-menu-close-line"></span>
+          </button>
+          <a class="burger-link" href="/pages/projects/projects.html">projects</a>
+          <a class="burger-link" href="/pages/members/members.html">members</a>
+          <a class="burger-link" href="/pages/about-us/about-us.html">about us</a>
+          <a class="burger-link" href="/pages/contact-us/contact-us.html">contact us</a>
+          <a class="burger-link" href="/pages/login/login.html">sign in</a>
+          <a class="burger-link" href="/pages/user-profile/user-profile.html">sign out</a>
```
Comment (dirk-jacobs) [inline client/components/header-component.js]: Sign out should not be visible if not signed in.
There should also a method be attached to clear the token

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
Overall looks good.
Signout / Sign In should be refined
For this, just check how it's done in the header and avatar

**Review by dirk-jacobs — APPROVED** [review]
This is a working version

**Comment by perezrei** [comment]
Good, I'll let everyone know that

**Review by Hajir-pro — APPROVED** [review]

**Pull request merged**